networklibrary/src/main/java/com/eis/communication/network/INetSubscribers.java

```diff
@@ -0,0 +1,31 @@
+package com.eis.communication.network;
+
+import com.eis.communication.Peer;
+
+import java.util.ArrayList;
+
+/**
+ * Interface which defines common operations for Subscribers of a network
```

Comment:
What exactly is a "Subscriber", what does it represent? It is not defined anywhere.